Various/ausbaxter_Render item columns.lua

```diff
@@ -0,0 +1,3878 @@
+--@description ausbaxter_Render item columns
```

Comment:
Please don't repeat the author name in the package name. It clutters the package list and breaks alphabetical sorting.

(Packages doing this currently will be edited in the future)